File: crm/src/verticals/eventsTicketing.ts
import { tab } from './shared';
import type { CrmVerticalDefinition } from './types';

export const eventsTicketingVertical: CrmVerticalDefinition = {
  key: 'events_ticketing',
  label: 'Events & Ticketing',
  riskLevel: 'low',
  entityLabelSingular: 'event',
  entityLabelPlural: 'events',
  defaultPlanLabel: 'Events plan',
  clientTabs: [
    tab('overview', 'Overview'),
    tab('readiness', 'Readiness'),
    tab('catalog', 'Events'),
    tab('bookings', 'Ticketing'),
    tab('leads', 'Waitlists/Leads'),
    tab('activity', 'Conversations'),
    tab('prompt', 'Prompt'),
    tab('controls', 'Controls'),
  ],
  entityTypes: ['event', 'venue', 'performer', 'ticket_type', 'showtime', 'organizer'],
  readinessChecks: ['events', 'date_location', 'ticket_handoff', 'venue_maps', 'policies', 'organizer_contact'],
};

File: agent/verticals/events_ticketing.py
"""Events and ticketing vertical definition."""

from agent.verticals.base import VerticalDefinition, tabs

VERTICAL = VerticalDefinition(
    key="events_ticketing",
    label="Events & Ticketing",
    risk_level="low",
    entity_label_singular="event",
    entity_label_plural="events",
    default_plan_label="Events plan",
    crm_tabs=tabs(
        ("overview", "Overview"),
        ("readiness", "Readiness"),
        ("catalog", "Events"),
        ("bookings", "Ticketing"),
        ("leads", "Waitlists/Leads"),
        ("activity", "Conversations"),
        ("prompt", "Prompt"),
        ("controls", "Controls"),
    ),
    entity_types=("event", "venue", "performer", "ticket_type", "showtime", "organizer"),
    readiness_checks=("events", "date_location", "ticket_handoff", "venue_maps", "policies", "organizer_contact"),
    action_types=("SHOW_ENTITIES", "CHECK_AVAILABILITY", "START_TICKET_PURCHASE", "JOIN_WAITLIST", "CAPTURE_LEAD"),
)
